transports/sql/native-delayed-delivery.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SQL Server Native Delayed Delivery
+summary: Describes the native delayed delivery implementation in the SQL Server transport
+reviewed: 2017-06-05
+component: SqlServer
+versions: '[3.1,)'
+---
+
+In Versions 3.1 and above, the SQL Server transport can take advantage of native [delayed delivery](/nservicebus/messaging/delayed-delivery.md) without then need to run the [timeout manager](/nservicebus/messaging/timeout-manager.md). Instead, the transport creates infrastructure which can delay messages using native SQL Server transport features. To enable the native delayed delivery use the following API:
+
+snippet: EnableNativeDelayedDelivery
+
+NOTE: In this mode the timeout manager will still be running in order to process all outstanding delayed messages. Refer to the [Disabling the timeout manager](/transports/sql/native-delayed-delivery.md#backwards-compatibility-disabling-the-timeout-manager) section for details on how to disable the timeout manager entirely.
+
+
+## Configuration
+
+The settings described in this section are obligatory. By default use the values presented in snippets. The settings can be fine-tuned to fit a particular system's characteristic, e.g. in case the expired timeout messages need to be picked up more frequently to result in a more precise approximation of expiry time.
```

Comment:
My mistake, MatureBatchSize and Suffix have defaults, I'll update.
@SzymonPobiega I may be wrong but I don't see where the default for interval is specified, although in the comment we say "Configures how often delayed messages are processed (every 5 seconds by default)."